src/main/java/frc/robot/commands/Drive.java

```diff
@@ -0,0 +1,60 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.filter.SlewRateLimiter;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Drivetrain;
+
+import java.util.function.Supplier;
+
+public class Drive extends Command {
+
+    private final double DRIVE_ACCELERATION_LIMIT = 2;
+    private final double TURN_ACCELERATION_LIMIT = 4;
+
+    private final Drivetrain drivetrain;
+    private final Supplier<Double> xSpeed;
+    private final Supplier<Double> ySpeed;
+    private final Supplier<Double> rotationSpeed;
+    private final boolean fieldRelative;
+    private final boolean usePID;
+
+    private final SlewRateLimiter xLimiter;
+    private final SlewRateLimiter yLimiter;
+    private final SlewRateLimiter rotationLimiter;
+    private final boolean limitAcceleration;
+
+    public Drive(Drivetrain drivetrain, Supplier<Double> xSpeed, Supplier<Double> ySpeed,
+                 Supplier<Double> rotationSpeed, boolean fieldRelative, boolean usePID,
+                 boolean limitAcceleration) {
+        addRequirements(drivetrain);
+        this.drivetrain = drivetrain;
+        this.xSpeed = xSpeed;
+        this.ySpeed = ySpeed;
+        this.rotationSpeed = rotationSpeed;
+        this.usePID = usePID;
+        this.limitAcceleration = limitAcceleration;
+        this.fieldRelative = fieldRelative;
```

Comment:
initialize these values in the same order as you receive them in the constructor